contributors/guide/reviewer-guidelines.md

```diff
@@ -0,0 +1,90 @@
+
+### Minor Changes
+
+If the PR is a minor change in the repository, you can browse through the changes line of codes through the GitHub UI. You can use the ["Split Diff View"](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-comparing-branches-in-pull-requests) for easy view of each LOC and also you can see changes made in the each commit of the patch using the UI.
+
+### Major Change:
+- If the PR is a major change which may affect a lot of section of your repository, You can pull the local changes of the PR into your local system and browse through each and every change made in the PR. You can also test & play around with the changes done in the PR to ensure each and every change is as expected.
+- To fetch the Pull Request locally, Use this -
+```sh
+$ git checkout -b `<pr-to-review>`
+$ git pull upstream/<PULL ID>/head
+```
+
+## Reviewing Ideology
+
+### Deciding what to review ?
+- As a Maintainer, you will might be completely overloaded with the Pending PRs left to review. Some of which will be from experienced contributors and some from the new contributor.
+- Since you know the experienced contributor already knows the guidelines for PR and Code to be written for repository. You should review the PR and patches from the experienced contributors first, as it will be less time consuming.
+- Until you have a large block of free time, You must put the newcomer PR for hold. Afterwhich, you should go line by line to the code changes in the PR and please avoid to criticse the `Code Structure`, `Coding Style` of the newcomer's PR.
+
+### Assigned Reviews
+- Maintainers can assign reviews to other maintainers, when appropriate. The
+assignee becomes the shepherd for that PR and is responsible for merging the PR
+once they are satisfied with it or else closing it. The assignee might request
+reviews from non-maintainers.
+
+### What to approve ?
+- Before approving the `Patch/PR`, Make sure :
+    + If it is a bug fix:
+        * Does the followign actually fixes the bug ?
+        * Is the bug real/worth fixing ?
+        * Is this the right fix ?
+    + If it is a feature request:
+        * Is the community okay with this feature ?
+        * Do we really need this feature to move forward?
+        * Does it break the other features in the repository?
+- If the scope of change is very large, It is always better to ask for smaller and focused PRs as they are much easy to interact and review.
+- Make sure the changes contain comments :
+    + If the patch does *anything* non-obvious, It needs to explain why and how with proper comments.
+    + If you made any decision while making some analysis, Explain why you chose the following and what basis.
+- Final note to reviewers, Code review time is exponential to the PR size, Hence it's better to check each change line by line so that it does not affect any other feature in the repository.
```

Comment:
```suggestion
- Final note to reviewers, code review time is exponential to the PR size, hence it's better to check each change line by line so that it does not affect any other feature in the repository.
```